Extension/src/LanguageServer/extension.ts

```diff
@@ -510,8 +582,8 @@ async function selectClient(): Promise<Client> {
     if (clients.Count === 1) {
         return clients.ActiveClient;
     } else {
-        const key: string = await ui.showWorkspaces(clients.Names);
-        if (key !== "") {
+        const key: string | undefined = await ui?.showWorkspaces(clients.Names);
```

Comment:
As mentioned previously, these `ui?`'s make me nervous, as these imply the behavior will change depending on execution timing, which is one of the worst types of bugs to investigate.  If we can await `getUI` in the first please, lets remove these `?`'s.

Reply:
I'm not sure how you mean

Reply:
Addressing it like how we discussed